docs/src/advanced-concepts/fine-tuning.rst

```diff
@@ -48,31 +48,17 @@ Prerequisites
 
 1. Train the Base Model. You can train the base model using the command:
 ``mtt train options.yaml``. Alternatively, you can use a pre-trained
-foundational model, if you have access to its state dict.
+foundational model, if you have access to its state dict. After this training,
```

Comment:
Its checkpoint? Some applications people might not know what a state dict is

Reply:
Thanks, I will change that!